server/routes/archiveGoal/mappers/archiveGoalFormToDtoMapper.test.ts

```diff
@@ -0,0 +1,24 @@
+import type { ArchiveGoalDto } from 'dto'
+import type { ArchiveGoalForm } from 'forms'
+import ReasonToArchiveGoalValue from '../../../enums/ReasonToArchiveGoalValue'
+import toArchiveGoalDto from './archiveGoalFormToDtoMapper'
+
+describe('archiveGoalFormToDtoMapper', () => {
+  it('should map from form to DTO object', () => {
+    const prisonNumber = 'FOO'
```

Comment:
As above re: using a prison number that is more representative of real data please